### DISCUSSION TITLE
Datetime change?

### DISCUSSION THREAD

**Discussion by andyburgessmd** [Budibase/budibase#10898] [Help]
Has anyone noticed a change to the datetime used in a budibase/internal DB since the last release?

I've had a table with filter for dates before today working fine until I've updated to the latest release.
The data table has a date field and then in the UI I have a filter where data is greater than or equal to a JS binding of
`const date = new Date();
date.setHours(00, 00, 00);
return date;`
so that it looks for all records from midnight today. If I output that to test I get `2023-06-14T00:00:00.821Z`.
But in the table block I now see results from many months ago that do not match the filter, the values returned in the JSON from the db query are `"date":"2022-08-11T00:00:00.000"` i.e. without the Z

Has anyone else noticed something different happening?

**Comment by melohagan** [comment]
Hey @andyburgessmd 

I wasn't able to replicate this. What timezone are you in? Are you able to provide any screenshots of your filters and table results, or even an app export?
